reactioncommerce/kinetic
# As the shop admin, I want to be able to control which parts of the UI a user can access

Resolves #51 
Testing Instructions:
- Start the app and log in with an admin permission account
- Navigate to Settings/Users and Permissions/Groups
- Select or create a group that you want to invite users to
- Navigate to Invitations tab, invite a user with your selected groups at the previous steps (make sure your shop has an email to invite users)
- Sign up a new user with the same email that you invited in the previous step
- Now check the group permission apply for hat user in store settings

Some screenshots:
<img width="1406" alt="image" src="https://user-images.githubusercontent.com/33818318/196932645-a175ae4a-396f-43b3-811e-7f99d7d0c06f.png">

<img width="1798" alt="Screen Shot 2022-10-20 at 18 04 18" src="https://user-images.githubusercontent.com/33818318/196932680-b0baa487-4715-43e4-902c-68a9899d4549.png">

If a user don't have shops/create permission and they don't have any invitation then after signing up, they should see this screen

<img width="1800" alt="Screen Shot 2022-10-20 at 18 09 13" src="https://user-images.githubusercontent.com/33818318/196933390-55b1c3a3-f884-457d-85ee-949f0dccd97e.png">
